Tasks/IpaResign/Tests/L0.ts

```diff
@@ -20,7 +20,7 @@ describe('ipa-resign L0 Suite', function () {
     });
     /* tslint:enable:no-empty */
 
```

Comment:
Could you please also fix test timeout here [like this](https://github.com/microsoft/app-store-vsts-extension/pull/204/commits/cc0c4bc1e3f9c5fff3649f781bc4a01e009175d6)? This task doesn't have any tests right now, but we may run into an issue with timeouts if we decide to add them later

Other than that, LGTM!

Reply:
Fixed separately in https://github.com/microsoft/app-store-vsts-extension/pull/212